src/core/components/TheWrapperDapp.vue

```diff
@@ -89,7 +89,7 @@ export default {
   methods: {
     onClose() {
       this.$router.push({
-        name: 'Dapps'
+        name: 'DApps'
```

Comment:
can you revert this back please? this is actual route name  and they all should be lower case